homeassistant/components/opentherm_gw/strings.json

```diff
@@ -11,10 +11,10 @@
       }
     },
     "error": {
-      "already_configured": "Gateway already configured",
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
       "id_exists": "Gateway id already exists",
-      "serial_error": "Error connecting to device",
-      "timeout": "Connection attempt timed out"
+      "serial_error": "[%key:common::config_flow::error::cannot_connect%]",
+      "timeout": "[%key:common::config_flow::error::cannot_connect%]"
```

Comment:
If we're going to use the same error for both cases, we may as well merge them into one `cannot_connect` error condition in the config flow.

Reply:
True, originally I did not want to touch the code, but I'll add a commit to reduce it to a single check.

Reply:
Ok, commited that.

Reply:
And I shouldn't have commited without running the tests again, so another commit to fix the test.